03-objects-and-composition/exercise/06.storeCatalogue.js

```diff
@@ -0,0 +1,16 @@
+function solve(arr) {
+
+    arr.sort((a, b) => a.localeCompare(b));
+
+    let currentLetter;
+
+    arr.forEach(el => {
+        if (el[0][0] != currentLetter) {
```

Comment:
I would suggest you to add more descriptive names to the variables and extract more items like el[0][0].

Reply:
@antoniaat,  Thanks for the advice.